src/main/java/com/dev/eficiente/nosso/bolao/core/validation/ValidationConfig.java

```diff
@@ -0,0 +1,17 @@
+package com.dev.eficiente.nosso.bolao.core.validation;
+
+import org.springframework.context.MessageSource;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.validation.beanvalidation.LocalValidatorFactoryBean;
+
+@Configuration
+public class ValidationConfig {

```

Comment:
*Usamos tudo que conhecemos que está pronto. Só fazemos código do zero se for estritamente necessário.*

A validação já vem configurada por default, não precisamos de nada extra :)